packit_service/worker/handlers/vm_image.py

```diff
@@ -135,7 +135,7 @@ def run(self) -> TaskResults:
             self.data._db_trigger = model.runs[0].get_trigger_object()
             status = self.data.event_dict["status"]
             model.set_status(status)
-            self.report_status(status, "")
+            self.report_status(status, self.data.event_dict["message"])
```

Comment:
wow, it's a oneliner, such a fundamental improvement to the workflow

Reply:
I hope this will work, I could not really test it 🤞